README.md

```diff
@@ -1,3 +1,7 @@
+Filer
+
+![Filer App](https://github.com/ayushmunot/Filer/blob/master/filer1.png?raw=true "filer")
```

Comment:
That's not a screenshot of Filer.
For the real screenshot, show Filer's main window and a "Edit rule" window nicely positioned. You'll have to take a screenshot of a full screen and crop etc. in an image editor (e.g. Wonderbrush). The background has to be transparent.
Join the IRC channel #haiku if you need help.